README.md

```diff
@@ -135,7 +135,7 @@ The list of resources will be displayed and you will be prompted to confirm the
 
 ## Custom connection targets
 
-Basti provides first class support for RDS instances and Aurora clusters. However, you can use Basti to connect to any other target in your AWS VPC (e.g. Elasticache instance, EC2 instance, etc.).
+Basti provides first class support for RDS instances, Aurora clusters, and Elasticache clusters. However, you can use Basti to connect to any other target in your AWS VPC (e.g. Elasticache instance, EC2 instance, etc.).
```

Comment:
remove the Elasticache mentions in the end (e.g. Elasticache instance, EC2 instance, etc.).